desktop/apps/artist/routes.coffee

```diff
@@ -40,7 +40,7 @@ sd = require('sharify').data
           res.locals.sd.ARTIST = artist
           res.locals.sd.TAB = tab
           res.locals.sd.CURRENT_ITEM = currentItem
-          res.locals.sd.ON_ARTIST_PAGE = true
+          res.locals.sd.ARTIST_PAGE_CTA_ENABLED = !(res.locals.sd.CURRENT_USER? || res.locals.sd.REFERRER?.includes(APP_URL))
```

Comment:
Since we want to customize some behavior w/ the follow button and sign up via header, but only when you were on the artist page with the CTA enabled, I figured this was a good place to add it to sharify.

Then, in a couple of places we can just do `return if ARTIST_PAGE_CTA_ENABLED` and what not.